recipe-server/normandy/base/tests/test_templatetags.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from normandy.base.templatetags.normandy_webpack_loader import render_bundle
+
+
+class TestRenderBundle(object):
+    @pytest.fixture
+    def js_file(self, tmpdir):
+        js_file = tmpdir.join('file.js')
+        js_file.write('console.log("test");')
+        return js_file
+
+    @pytest.fixture
+    def css_file(self, tmpdir):
+        css_file = tmpdir.join('file.css')
+        css_file.write('html { padding: 10px; }')
+        return css_file
+
+    @pytest.fixture
+    def mock_get_files(self, mocker, js_file, css_file):
+        get_files = mocker.patch('normandy.base.templatetags.normandy_webpack_loader.get_files')
+        get_files.return_value = [
+            {'path': js_file.strpath, 'name': 'file.js', 'url': 'file.js'},
+            {'path': css_file.strpath, 'name': 'file.css', 'url': 'file.css'},
+        ]
+        return get_files
+
+    @pytest.fixture
+    def mock_sri_hash(self, mocker):
+        sri_hash = mocker.patch('normandy.base.templatetags.normandy_webpack_loader.sri_hash')
+        sri_hash.return_value = 'mock-hash'
```

Comment:
I'd rather see this unmocked, or at least use a return value that depends on the inputs so that the tests can assert that the right hash goes with the right file. Maybe something as simple as

```python
def mock_sri_hash(contents):
    return 'hash({})'.format(contents)
```